src/aspire/utils/bot_align.py

```diff
@@ -0,0 +1,181 @@
+"""
+This code is an ASPIRE port of Ruiyi Yang's `Bayesian Optimal Transport Alignment`.
+
+`https://github.com/RuiyiYang/BOTalign`
+
+`https://arxiv.org/pdf/2305.12310.pdf`
+"""
+
+import numpy as np
+import pymanopt
+from numpy.linalg import norm
+from scipy.optimize import minimize
+
+from aspire.operators import wemd_embed
+from aspire.utils.rotation import Rotation
+
+
+def align_BO(
+    vol_ref,
+    vol_given,
+    loss_type="wemd",
+    downsampling_level=32,
+    max_iters=200,
+    refine=True,
+    tau=1e-3,
+    man_max_iter=500,
```

Comment:
Why `man`? Maybe `surrogate` would be better here?

Reply:
man was just short for (py)man opt.  Replaced with surrogate.

Reply:
Ok got it. Thanks.